packages/fxa-shared/subscriptions/types.ts

```diff
@@ -156,3 +156,7 @@ export const SubscriptionUpdateEligibility = {
 
 export type SubscriptionUpdateEligibility =
   typeof SubscriptionUpdateEligibility[keyof typeof SubscriptionUpdateEligibility];
+
+export enum SubscriptionStripeErrorType {
+  NO_MIN_AMOUNT_FOR_CURRENCY = 'Currency does not have a minimum amount available.',
+}
```

Comment:
Have you considered extending `Error` to give these error a different type?

Reply:
Not a blocker.  But I was thinking that we could use `instanceOf` to check for the type of error.  `SubscriptionStripeErrorType` seems too generic for that purpose.  E.g. `new StripeNoMinChargeCurrencyError('de')`.  Something like that.